frontend/src/i18n/i18n.js

```diff
@@ -54,12 +45,12 @@ const resources = {
 i18n
   .init({
     resources,
-    lng: `${language}_${currentSyndicate}`,
-    fallbackLng: `${language}_base`,
+    lng: `${currentLanguage}_${currentSyndicate}`,
+    fallbackLng: `${currentLanguage}_base`,
     interpolation: {
       escapeValue: false,
     },
-    debug: true,
+    debug: false,
```

Comment:
We should leave debug "true" so that it'll prompt us when we miss a key in the translation files.

Reply:
All keys are missing